Update version range for Liberty install to newer version

The current version range is specified as [23.0.0.3,) which is quite old. A user recently reported seeing this error when trying to create a server in Liberty Maven plugin.

```
[ERROR] Failed to execute goal io.openliberty.tools:liberty-maven-plugin:3.11.2:install-server (prepare-package) on project dv-caching: Could not get the highest version from the range: [22.0.0.3,): Unable to resolve version range from io.openliberty:openliberty-kernel:zip:[22.0.0.3,) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.openliberty.tools:liberty-maven-plugin:3.11.2:install-server (prepare-package) on project dv-caching: Could not get the highest version from the range: [22.0.0.3,)
```

It would be good to update the first version to something recent like 25.0.0.2 (or 25.0.0.3 which is scheduled to GA next week).

The code with the range defined is in `InstallLibertyTask` in a variable `defaultRuntime`.

--- a/.github/workflows/gradle.yml
+++ b/.github/workflows/gradle.yml
@@ -18,7 +18,7 @@ jobs:
       matrix:
         # test against latest update of each major Java version, as well as specific updates of LTS versions:
         RUNTIME: [ol, wlp]
-        RUNTIME_VERSION: [24.0.0.12]
+        RUNTIME_VERSION: [25.0.0.3]
         java: [21, 17, 11, 8]
         exclude:
         - java: 8
@@ -100,7 +100,7 @@ jobs:
       matrix:
         # test against latest update of each major Java version, as well as specific updates of LTS versions:
         RUNTIME: [ol, wlp]
-        RUNTIME_VERSION: [24.0.0.12]
+        RUNTIME_VERSION: [25.0.0.3]
         java: [21, 17, 11, 8]
         exclude:
         - java: 8

--- a/README.md
+++ b/README.md
@@ -101,7 +101,7 @@ Example using `libertyRuntime` property to install an Open Liberty beta runtime:
 
 ```groovy
 dependencies {
-    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '23.0.0.2-beta'
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '25.0.0.2-beta'
 }
 ```
 
@@ -109,7 +109,7 @@ Example using `libertyRuntime` property to install a specific Open Liberty runti
 
 ```groovy
 dependencies {
-    libertyRuntime group: 'io.openliberty', name: 'openliberty-kernel', version: '23.0.0.3'
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-kernel', version: '25.0.0.3'
 }
 ```
 
@@ -119,7 +119,7 @@ Example using the `libertyRuntime` property to install a WebSphere Liberty runti
 
 ```groovy
 dependencies {
-    libertyRuntime group: 'com.ibm.websphere.appserver.runtime', name: 'wlp-webProfile8', version: '22.0.0.12'
+    libertyRuntime group: 'com.ibm.websphere.appserver.runtime', name: 'wlp-webProfile8', version: '24.0.0.12'
 }
 ```
 

--- a/src/main/groovy/io/openliberty/tools/gradle/tasks/InstallLibertyTask.groovy
+++ b/src/main/groovy/io/openliberty/tools/gradle/tasks/InstallLibertyTask.groovy
@@ -38,7 +38,7 @@ class InstallLibertyTask extends AbstractLibertyTask {
     protected String detachedCoords
     protected String detachedConfigFilePath
     // default to install the latest Open Liberty kernel from Maven Central repository
-    protected String defaultRuntime = "io.openliberty:openliberty-kernel:[23.0.0.3,)"
+    protected String defaultRuntime = "io.openliberty:openliberty-kernel:[25.0.0.3,)"
 
     InstallLibertyTask() {
         configure({

